SwanCustomEnvironments/swancustomenvironments/serverextension.py

```diff
@@ -0,0 +1,83 @@
+from tornado import web
+from traitlets import Unicode
+from traitlets.config import Configurable
+
+from jupyter_server.base.handlers import JupyterHandler, APIHandler
+from jupyter_server.utils import url_path_join
+
+import subprocess
+from os import path
+
+
+class SwanCustomEnvironments(Configurable):
+    """General-purpose static configuration options that evolve the API for creating customized environments"""
+
+    makenv_path = Unicode(
+        path.join(path.dirname(__file__), "scripts/makenv.sh"),
+        config=True,
+        help='Path to the script used to create custom environments.'
+    )
+
+
+class SwanCustomEnvironmentsApiHandler(APIHandler):
+    """
+    API handler for creating custom environments
+    Runs a local script to create a custom environment and streams the output to the client
+    Environment name and repository URL are passed as query arguments and are mandatory
+    ACCPy version is optional (if not provided, generic python is used)
```

Comment:
I would remove this from here. It can easily get outdated. I would add an explanation about the arguments received to a docstring of the `get` method, to have the docs closer to where things happen.